### PR TITLE
Circe 0.7.0

### PR DESCRIPTION
**Pull request by tomrf1**
and use the latest fezziwig

### CODE REVIEW

**Comment by shtukas** [comment]
:+1:

File: json/src/test/scala/com/gu/contentapi/json/utils/JsonHelpers.scala [hunk 1]
```diff
@@ -14,7 +14,7 @@ object JsonHelpers {
 
   def parseJson[T : Decoder](rawJson: String): T = {
     val json = parse(rawJson).leftMap(e => throw e).getOrElse(Json.Null)
-    val response = json.cursor.downField("response").map(c => c.focus).getOrElse(Json.Null)
+    val response = json.hcursor.downField("response").top.getOrElse(Json.Null)
```
Comment (LATaylor-guardian) [inline json/src/test/scala/com/gu/contentapi/json/utils/JsonHelpers.scala]: Why have we switched the type of the cursor?

Comment (tomrf1) [inline json/src/test/scala/com/gu/contentapi/json/utils/JsonHelpers.scala]: You now only get the hcursor, see https://github.com/circe/circe/pull/459

**Comment by LATaylor-guardian** [comment]
👍

File: json/src/test/scala/com/gu/contentapi/json/utils/JsonHelpers.scala (hunk 1 above)
Comment (mchv) [inline json/src/test/scala/com/gu/contentapi/json/utils/JsonHelpers.scala]: Did you run the benchmarks to see if we saw a perf improvements in our context?

Comment (tomrf1) [inline json/src/test/scala/com/gu/contentapi/json/utils/JsonHelpers.scala]: @mchv good question, I'll take a look

**Comment by tomrf1** [comment]
@mchv I ran the benchmarks for master (with circe 0.6.1) and this branch (circe 0.7.0). It hasn't got any better...

**0.6.1**
```
[info] # Run complete. Total time: 00:29:55
[info]
[info] Benchmark                                          Mode  Cnt        Score        Error  Units
[info] JsonDecodeBenchmark.circeDecode                    avgt  200   128292.072 ±  13797.247  ns/op
[info] JsonDecodeBenchmark.circeDecodeEachContent         avgt  200  1502412.577 ± 211224.354  ns/op
[info] JsonDecodeBenchmark.circeDecodeEachMassiveContent  avgt  200      311.056 ±      3.758  ms/op
[info] JsonDecodeBenchmark.circeEncode                    avgt  200   127464.974 ±   2623.759  ns/op
```

**0.7.0**
```
[info] # Run complete. Total time: 00:29:43
[info]
[info] Benchmark                                          Mode  Cnt        Score        Error  Units
[info] JsonDecodeBenchmark.circeDecode                    avgt  200   140537.775 ±   6763.709  ns/op
[info] JsonDecodeBenchmark.circeDecodeEachContent         avgt  200  1372619.532 ± 108916.797  ns/op
[info] JsonDecodeBenchmark.circeDecodeEachMassiveContent    avgt  200      375.148 ±     16.611  ms/op
[info] JsonDecodeBenchmark.circeEncode                    avgt  200   133315.055 ±   5111.019  ns/op
```

**Pull request merged**